.gitignore

```diff
@@ -2,4 +2,6 @@
 /node_modules
 /npm-debug.log
 /tmp
-/build
+/lib
```

Comment:
`/lib` is no longer in the project (although you may want to keep it based on my comments below)